hooks/post/30-upgrade.rb

```diff
@@ -38,15 +38,13 @@ def import_subscriptions
   Kafo::Helpers.execute('foreman-rake katello:upgrades:2.4:import_subscriptions')
 end
 
-def remove_elasticsearch
+def elasticsearch_message
   return true unless Kafo::Helpers.execute('rpm -q elasticsearch')
 
   gems = ['ruby193-rubygem-tire', 'tfm-rubygem-tire', 'elasticsearch', 'sigar', 'snappy-java', 'lucene4-contrib', 'lucene4']
-  gems.each do |gem|
-    Kafo::Helpers.execute("rpm -e #{gem}")
-  end
-  message = "Elasticsearch has been removed as a dependency, the database files can be "\
-            "removed manually with #rm -rf /var/lib/elasticsearch"
+  message = "Elasticsearch has been removed as a dependency.  The database files can be "\
+            "removed manually with #rm -rf /var/lib/elasticsearch.  "
+  message += "Some packages are no longer needed and can be removed:  #rpm -e #{rpms.join(' ')}"
```

Comment:
Is it worth doing a check for the packages and /var/lib/elasticsearch and if none of them are found, suppress the message or parts therein?